### DISCUSSION TITLE
Why doesn't UniGetUI not LIST ALL the Apps Available to download, and also showing which Source it's from?

### DISCUSSION THREAD

**Discussion by leeuniverse** [Devolutions/UniGetUI#2958] [Q&A] [answered]
This is really needed....  That way we have one tool to find and keep updated all the available software.
We STILL have to download all the various distributions to see what we might want.

The Search is only useful if we already know what app we want, so it's not as useful because part of the point of these distributions is to see various tools that we might want etc.

Please add....

**Comment by marticliment** [comment, marked as answer]
This is not a thing anymore because with the current amount of available packages you would be listing around 2.000.000 packages, and your PC would go kaboom. Furthermore, most of the package managers don't even allow to obtain the full list of packages for a manager

**Comment by leeuniverse** [comment]
Ya, that makes sense....  I was looking at some of the distro's and ya, there are a LOT of apps now in them.
I guess it's no longer I remember years back when you could go to a website, some program allowed you to install lot's of apps, they would be listed, but I guess with this type that's just not doable anymore.
